test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -433,28 +433,50 @@ public void execute_find_isCaseSensitive() throws Exception {
         assertCommandBehavior("find KEY",
                                 Command.getMessageForPersonListShownSummary(expectedList),
                                 expectedAB,
-                                true,
+                                 true,
                                 expectedList);
     }
 
-    @Test
+     @Test
     public void execute_find_matchesIfAnyKeywordPresent() throws Exception {
         TestDataHelper helper = new TestDataHelper();
         Person pTarget1 = helper.generatePersonWithName("bla bla KEY bla");
         Person pTarget2 = helper.generatePersonWithName("bla rAnDoM bla bceofeia");
         Person p1 = helper.generatePersonWithName("key key");
         Person p2 = helper.generatePersonWithName("KEy sduauo");
+        Person pTarget3 = helper.generatePersonWithName("12345678");
+        Person pTarget4 = helper.generatePersonWithName("87654321");
```

Comment:
These 2 lines create Person objects with the argument becoming the **name**, not the phone number.

Reply:
Also, your test should be in a separate function because it tests a separate functionality from yours.